src/snowcli/cli/snowpark_shared.py

```diff
@@ -52,8 +52,8 @@ def snowpark_create(
     env_conf = AppConfig().config.get(environment)
     validate_configuration(env_conf, environment)
     if type == "function" and install_coverage_wrapper:
-        print(
-            """You cannot install a code coverage wrapper on a function, only a procedure."""
+        log.error(
+            "You cannot install a code coverage wrapper on a function, only a procedure."
         )
         raise typer.Abort()
```

Comment:
Can you show example how this will look like in cli?

Reply:
```% snow login
Path to Snowsql config [~/.snowsql/config]: config
Connection name (for entry in snowsql config): connection
2023-06-20 14:43:39 ERROR Path to snowsql config does not exist: config
Aborted.
```